bench/plutus-scripts-bench/src/Cardano/Benchmarking/ScriptAPI.hs

```diff
@@ -18,3 +20,14 @@ data PlutusBenchScript
 
 mkPlutusBenchScript :: String -> ScriptInAnyLang -> PlutusBenchScript
 mkPlutusBenchScript = PlutusBenchScript
+
+prepareScriptName :: String -> String
+prepareScriptName
+  = head
+  . dropWhile (=="hs")
+  . map unpack
+  . reverse
+  . split (=='.')
+  . last
+  . split (=='/')
+  . pack
```

Comment:
https://hackage.haskell.org/package/filepath-1.4.100.3/docs/System-FilePath.html#v:dropExtension can simplify this a lot.